_format/1.1/index.md

```diff
@@ -1916,32 +1927,38 @@ parameter **MUST** equal a space-separated (U+0020 SPACE, " ") list of profile U
 > (U+0022 QUOTATION MARK, "\"") if it contains more than one URI.
 
 A client **MAY** use the `profile` media type parameter in conjunction with the
-JSON:API media type in an `Accept` header to _request_, but not _require_, that
-the server apply one or more profiles to the response document. When such a
-request is received, a server **SHOULD** attempt to apply the requested profiles
-to its response.
+JSON:API media type in an `Accept` header to request that the server apply one
+or more profiles to the response document. When such a request is received, a
+server **SHOULD** attempt to apply the requested profiles to its response.
```

Comment:
I understand why this is saying
> **SHOULD** attempt to apply …

But I think that
> **MUST** either apply or respond with a 406*

would be more explicit, more clear. I am probably overlooking the reason(s) why we cannot do that. But that would definitely make profiles a lot simpler to understand, which is the primary goal here.

Curious to hear your feedback 😀